data/src/main/kotlin/com/antosik/benchproject/data/popular/movies/entity/remote/MovieResponseRemote.kt

```diff
@@ -1,19 +1,10 @@
 package com.antosik.benchproject.data.popular.movies.entity.remote
 
 import com.antosik.benchproject.data.movies.common.constants.MoviesConstants
-import com.antosik.benchproject.data.popular.movies.entity.database.MovieEntity
 
 internal data class MovieResponseRemote(
     val results: List<MovieRemote>,
-) {
-    fun toEntity() = results.map {
-        MovieEntity(
-            it.id,
-            it.name,
-            it.rating,
-            it.releaseDate,
-            MoviesConstants.BASE_URL_FOR_IMAGE + it.imagePath,
-            false
-        )
-    }
-}
+)
+
+internal fun String.urlPrefix() =
```

Comment:
i would go with doing this private in the place we need OR just use it, without fun extraction in the place we need this. We should try to avoid primitive types ext. if we can :D

Reply:
I will delete it and I just go the old way. So I will add this prefix in mapper 😄